src/Signin.tsx

```diff
@@ -98,6 +103,15 @@ export class Signin extends PureComponent<Props, State> {
 
       clearInterval(this.intervalID)
 
+      if (CLOUD && isFlagEnabled('authSessionCookieOn')) {
+        const url = new URL(
+          `${window.location.origin}${CLOUD_LOGIN_PATHNAME}?redirectTo=${window.location.href}`
+        )
+        setToLocalStorage('redirectTo', window.location.href)
+        window.location.href = url.href
+        throw error
```

Comment:
sorry if this is a dumb question - why are we throwing an error here?

Reply:
Not a dumb question - based on the previous convention of having a stopping operation if the user isn't redirected. Instead of plopping a `return` here (which would do the same thing), we would want to throw an error in the hopes that we can gather information on what happened that a simple redirect would go wrong

Reply:
got it - but in normal scenarios, the user is redirected before we get to this line so no error is thrown?

Reply:
yup, that's the hope 🤞🏽